build-aux/snap/snapcraft.yaml

```diff
@@ -191,13 +191,30 @@ parts:
       craftctl default
       python3 "${CRAFT_PROJECT_DIR}/build-aux/snap/local/patch-dl.py" "/snap/snapd/current/usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/${DYNAMIC_LINKER}"
 
+  patchelf:
+    plugin: autotools
+    build-packages:
+      - autoconf-archive
+      - bison
+      - flex
+      - gettext
+      - g++
+      - pkg-config
+      - wget
+    source: https://github.com/NixOS/patchelf/releases/download/0.18.0/patchelf-0.18.0.tar.bz2
```

Comment:
Could we use the Ubuntu or Debian archive here?

http://archive.ubuntu.com/ubuntu/pool/universe/p/patchelf/patchelf_0.18.0.orig.tar.gz

Debian applies two patches that we should consider as well.

Reply:
done